kaiax/auction/impl/bid_pool_test.go

```diff
@@ -222,6 +222,54 @@ func TestBidPool_AddBid(t *testing.T) {
 	assert.Equal(t, auction.ErrLowBid, err)
 }
 
+func TestBidPool_AddBid_MaxBidPoolSize(t *testing.T) {
```

Comment:
Can we make it shorter to reduce the scope of test? (line 268-270 seems only effective)